src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -982,14 +982,30 @@ bool isLambdaConversion(BoundExpression expr)
                 Debug.Assert(lastParam.IsParams);
 
                 // Create an empty array for omitted param array argument.
-                BoundExpression argument = CreateParamArrayArgument(syntax, lastParam.Type, ImmutableArray<BoundExpression>.Empty, compilation, localRewriter: null);
+                BoundExpression argument = createParamArrayArgument(syntax, methodOrIndexer, lastParam.Type, ImmutableArray<BoundExpression>.Empty, compilation);
                 ArgumentKind kind = ArgumentKind.ParamArray;
 
                 argumentsInEvaluationBuilder.Add(operationFactory.CreateArgumentOperation(kind, lastParam.GetPublicSymbol(), argument));
             }
 
             Debug.Assert(argumentsInEvaluationBuilder.All(static arg => arg is not null));
             return argumentsInEvaluationBuilder.ToImmutableAndFree();
+
+            static BoundExpression createParamArrayArgument(
+                SyntaxNode syntax,
+                Symbol methodOrIndexer,
+                TypeSymbol parameterType,
+                ImmutableArray<BoundExpression> arrayArgs,
+                CSharpCompilation compilation)
+            {
+                var method = methodOrIndexer switch
+                {
+                    MethodSymbol m => m,
+                    PropertySymbol p => p.GetOwnOrInheritedGetMethod() ?? p.GetOwnOrInheritedSetMethod(),
+                    _ => throw ExceptionUtilities.UnexpectedValue(methodOrIndexer),
+                };
+                return CreateParamArrayArgument(syntax, parameterType, arrayArgs, compilation, method, BindingDiagnosticBag.Discarded);
```

Comment:
>method [](http://example.com/codeflow?start=95&length=6)

I am not sure what we are doing here, the value in this local doesn't look like a containing method.